### PR TITLE
community: Add PHP language parser to document_loaders

### PR DESCRIPTION
**Pull request by david02871**
**Description:**
Added a PHP language parser to document_loaders
**Issue:** N/A
**Dependencies:** N/A
**Twitter handle:** N/A

### CODE REVIEW

File: libs/community/langchain_community/document_loaders/parsers/language/php.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        (namespace_definition) @namespace
+    ]
+""".strip()
+
+
+class PHPSegmenter(TreeSitterSegmenter):
+    """Code segmenter for PHP."""
+
+    def get_language(self) -> "Language":
+        from tree_sitter_languages import get_language
```
Comment (ccurme) [inline libs/community/langchain_community/document_loaders/parsers/language/php.py:27]: Would be nice to add a try/except here with a nice error message, similar to https://github.com/langchain-ai/langchain/blob/356a139b0a6c1ac5a0eff95c265ce226a91838bb/libs/community/langchain_community/document_transformers/html2text.py#L31

**Comment by ccurme** [comment]
Thanks, looks great! Would you mind adding an example to the docs here? https://github.com/langchain-ai/langchain/blob/master/docs/docs/modules/data_connection/document_transformers/code_splitter.ipynb

**Review by ccurme — APPROVED** [review]

File: libs/community/langchain_community/document_loaders/parsers/language/php.py (hunk 1 above)
Comment (ccurme) [reply libs/community/langchain_community/document_loaders/parsers/language/php.py:27]: ah see this is covered in TreeSitterSegmenter.

**Comment by david02871** [comment]
@ccurme Thanks for your help and for taking a look at this!

**Comment by ccurme** [comment]
> @ccurme Thanks for your help and for taking a look at this!

No, thank you!

**Pull request merged**